project/Game.py

```diff
@@ -137,7 +149,15 @@ def quit_question(self, team, password):
         return True
 
     def answer_question(self, team, answer):
-        return True
+        if self.game.landmarks[team.currentLandmark].answer is not answer:
+            self.team.add_penalty()
```

Comment:
there is no self.team. maybe you just mean team? The parameter?

Reply:
Its been a long day.  Yeah I updated it.